core/domain/exp_domain.py

```diff
@@ -2499,3 +2499,470 @@ def add_contribution_by_user(self, contributor_id):
+                else:
+                    state_names_of_renamed_states[new_state_name] = (
+                        old_state_name)
+                if (state_names_of_renamed_states[new_state_name] not in
+                        old_to_new_state_names):
+                    change_is_mergeable = True
+            elif change.cmd == CMD_EDIT_STATE_PROPERTY:
+                state_name = change.state_name
+                if change.state_name in state_names_of_renamed_states:
+                    state_name = (
+                        state_names_of_renamed_states[change.state_name])
+                if state_name in old_to_new_state_names:
+                    # Here we will send the changelist, frontend_version,
+                    # backend_version and exploration to the admin, so
+                    # that the changes related to state renames can be
+                    # reviewed and the proper conditions can be written
+                    # to handle those cases.
+                    return False, True
+                if state_name not in self.changed_translations:
+                    self.changed_translations[state_name] = []
+                old_exp_states = (
+                    exp_at_change_list_version.states[state_name])
+                current_exp_states = (
+                    current_exploration.states[state_name])
+                if (change.property_name ==
+                        STATE_PROPERTY_CONTENT):
+                    if state_name in self.changed_properties:
+                        if (old_exp_states.content.html ==
+                                current_exp_states.content.html):
+                            if (STATE_PROPERTY_CONTENT not in
+                                    self.changed_translations[state_name] and
+                                    STATE_PROPERTY_RECORDED_VOICEOVERS not in
+                                    self.changed_properties[state_name]):
+                                change_is_mergeable = True
+                    else:
+                        change_is_mergeable = True
+                elif (change.property_name ==
+                      STATE_PROPERTY_INTERACTION_ID):
+                    if state_name in self.changed_properties:
+                        if (old_exp_states.interaction.id ==
+                                current_exp_states.interaction.id):
+                            if (STATE_PROPERTY_INTERACTION_CUST_ARGS
+                                    not in
+                                    self.changed_properties[state_name] and
+                                    STATE_PROPERTY_INTERACTION_ANSWER_GROUPS
+                                    not in
+                                    self.changed_properties[state_name] and
+                                    STATE_PROPERTY_INTERACTION_SOLUTION
+                                    not in
+                                    self.changed_properties[state_name]):
```

Comment:
Why aren't we having PROPERTY_CONFLICTING_INTERACTION_ID_CHANGE?

Reply:
Added.